lib/components/molecules/AccordionBasic/AccordionBasic.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+/**
+ *
+ * AccordionBasic
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import type { Node } from 'react';
+import styles from './AccordionBasic.style';
+
+type Props = {
+  className?: string,
+  title?: string,
+  value?: string,
+  isOpen?: boolean,
+};
+
+const AccordionBasic = ({ className, title, value, isOpen }: Props): Node => (
+  	<li className={className}>
+		<details className="accordion-item" open={isOpen}>
+			<summary className="accordion-item_heading">
+				{title}
+			</summary>
+			<p>{value}</p>
```

Comment:
Will this value be able to hold other components? Or Accordion component should only work with text?